ReadMe.md

```diff
@@ -58,7 +58,10 @@ To edit | 編集（する） | へんしゅう |
 Automatic | 自動 | じどう |
 Screen | 画面 | がめん |
 Video | 動画 | どうが |
+Picture | 写真 |　しゃしん
```

Comment:
Doesn't this rather mean "photo"? Do people use this when referring to images that are not shot by a camera?